masschallenge/django-accelerator
# AC-7818 Add 'Meeting Information' to Office Hours model

### Changes introduced in: [AC-7818](https://masschallenge.atlassian.net/browse/AC-7818):
- Add a migration to add meeting_info
### How to test
- Checkout this branch
- On accelerate `make migrate`
- Check out this branch too on impact and run `make runserver`
- Schedule office hours as a mentor and and navigate to this [api endpoint](http://localhost:8000/api/v1/office_hours_calendar/)
- Notice the new field `meeting_info`
### This PR should merge before the one on impact
